src/accelerate/utils/other.py

```diff
@@ -256,7 +256,7 @@ def load(f, map_location=None, **kwargs):
             torch.serialization.add_safe_globals(TORCH_SAFE_GLOBALS)
         else:
             kwargs.pop("weights_only", None)
-        loaded_obj = torch.load(f, map_location=map_location, **kwargs)
+        loaded_obj = torch.load(f, map_location=map_location, **kwargs, weights_only=True)
```

Comment:
Some tests are failing with

> TypeError: torch.serialization.load() got multiple values for keyword argument 'weights_only'

Probably, `weights_only` needs to be added to `kwargs` here.